### PR TITLE
Custom email guide: add table of supported languages

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds table of default supported languages with BCP 47 format.
- **Is this PR related to a Monolith release?** Nope.

### Resolves:

* [OKTA-735376](https://oktainc.atlassian.net/browse/OKTA-735376)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/c04152cad5161c8efbab1b8627bfc5902c17cdf1/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a22eeab1-356e-4fa2-a933-5f1e8c3f6914) | 71  | 92  | 98  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.27fea7f1-90c8-480b-9a29-c57fc6f3c238?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/070a35cc8229655142704747101b8cf3a5b60cd9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8174c341-68b0-4b90-94a6-513037c5eb5a) | 72  | 92  | 98  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4c85dc81-a612-4396-a8ff-19037a0f4d03?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ See [Email template operations](/docs/reference/api/brands/#email-template-opera
 
 Use these steps to add or edit a template in one of the Okta-supported languages.
 
-> **Note:** To access email customization with a free developer edition of an Okta org, you need to contact [Okta support](https://support.okta.com/help).
+> **Note:** To access email customization with a free Okta Developer Edition org, you need to contact [Okta support](https://support.okta.com/help).
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:82]:
```suggestion
> **Note:** To access email customization with a free Okta Developer Edition org, contact [Okta support](https://support.okta.com/help).
```

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 2]
```diff
@@ -187,6 +187,38 @@ After you customize a template in one language, you need to customize templates
...
+| Portuguese  | `pt-BR` |
+| Romanian  | `ro` |
+| Russian  | `ru` |
+| Swedish  | `sv` |
+| Thai  | `th` |
+| Turkish  | `tr` |
+| Ukrainian  | `uk` |
+| Vietnamese  | `vi` |
+| Chinese (simplified)  | `zh-CN` |
+| Chinese (traditional)  | `zh-TW` |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:220]: Should Chinese (simplified) and Chinese (traditional) go at the top of the list so they're alphabetical?

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/9a92d31b18c14aba574f322d82eca95351fba0f3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/280ada86-bb9b-4f29-a032-8536ca67264c) | 72  | 92  | 98  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.6758eb95-6cff-4846-aa85-30e79428bc60?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md (hunk 2 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:220]: Nice one. Done :)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/00dfebdbf71816a2031ecb9facc5e06eb738725f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9b1c996e-1beb-4f02-90f5-06be6eaeb333) | 72  | 92  | 98  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.65565ee7-279a-43ae-924b-50edd5b850ef?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**